src/test/java/support/fakes/FakeOkapi.java

```diff
@@ -178,4 +179,12 @@ private void registerFakeContributorNameTypesModule(Router router) {
       .withCollectionPropertyName("contributorNameTypes")
       .create().register(router);
   }
+
+  private void registerFakeUsersModule(Router router) {
```

Comment:
The new dependency on the user’s interface needs expressing in the Module Descriptor requires section.

Reply:
done